fastcrypto/src/polynomial.rs

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+// TODO: add license notice
```

Comment:
Notice re what?

Reply:
Added a reference to https://github.com/celo-org/celo-threshold-bls-rs (also Apache-2.0).
Let me know if more clarifications are needed.